mock-server/src/strimzi-api-handlers.js

```diff
@@ -1,120 +1,143 @@
-var topics = require('../_data_/topics.json')
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+var topics = require('../_data_/topics.json');
+
+const disableCORS = (res) => {
```

Comment:
We can use `api.use(cors())` in `mock-server/src/index.js` for less code.

Reply:
Thanks will make the update